### PR TITLE
Add terms to Terminology Section

### LINKED ISSUE #8: add terms to terminology section

### PR DESCRIPTION
**Pull request by jischr**
PR closes #8 

I added a first round of terms that would be useful to a reader. Feel free to make an issue to add more terms that you think are required!

### CODE REVIEW

File: spec/spec.md [hunk 1]
```diff
@@ -82,9 +82,60 @@ A description to the reader on how the document is structured.
...
 
-Note: the term Verifier in the [[ref: VC Data Model]] corresponds to a term Relying Party (RP) in [[ref:SIOPv2]].
+[[def:Authorization Response]]
+~ OAuth 2.0 Authorization Response extended by [[ref:OIDC]] and [[ref:OpenID4VP]]
+
+[[def:Claim, Claims]]
+~ An assertion made about a subject
+
+[[def:Decentralized Identifier, DID]]
+~ An identifier that enables verifiable, decentralized digital identity as defined by the W3C specification
```
Comment (Sakurann) [inline spec/spec.md:95]: should be `identifier`. DIDs are not identity - it's just a way to get pubKeys by reference

**Comment by jischr** [comment]
So someone does not need to click into the other links to figure out what the words referenced in this profile mean. Most all specs do this. there is a lot of repetition of terminology across specs i've read. usually they are defined with a "twist" as it relates to this profile.

**Comment by Sakurann** [comment]
the profile is not redefining these terms. if there is a "twist" in terminology, there is a problem

**Comment by jischr** [comment]
"Twist" might have been the wrong word here. it should define the term with an emphasis on how it relates to this profile. For example. An OpenID Provider in _this context_ is OAuth_ 2.0 Authentication Server implementing OIDC and OpenID4VP. In other contexts, it would just be an auth service implementing OIDC....

**Comment by Sakurann** [comment]
Thanks for the explanation. Yeah, I guess harmonizing terminology across specifications from various SDOs is a good thing

File: spec/spec.md [hunk 2]
```diff
@@ -82,9 +82,60 @@ A description to the reader on how the document is structured.
...
-Common terms in this document come from the specifications listed in (#reference).
+[[def:Authorization Request]]
+~ OAuth 2.0 Authorization Request extended by [[ref:OIDC]] and [[ref:OpenID4VP]]
 
-Note: the term Verifier in the [[ref: VC Data Model]] corresponds to a term Relying Party (RP) in [[ref:SIOPv2]].
+[[def:Authorization Response]]
+~ OAuth 2.0 Authorization Response extended by [[ref:OIDC]] and [[ref:OpenID4VP]]
+
+[[def:Claim, Claims]]
+~ An assertion made about a subject
```
Comment (Sakurann) [inline spec/spec.md:92]: `about an entity` or `about a subject of a credential`

File: spec/spec.md [hunk 3]
```diff
@@ -82,9 +82,60 @@ A description to the reader on how the document is structured.
 
 ## Terminology
```
Comment (Sakurann) [inline spec/spec.md:83]: I think I would appreciate a sentence saying something along the lines of "terms are defined in each specification used in this profile and this section bring them together for readability".

File: spec/spec.md (hunk 1 above)
Comment (Sakurann) [reply spec/spec.md:95]: for the purpose of this section I am more comfortable with clarifying that the core property of DIDs is the ability to enable Clients to obtain key material and other metadata by reference.

File: spec/spec.md [hunk 4]
```diff
@@ -82,9 +82,60 @@ A description to the reader on how the document is structured.
...
-Note: the term Verifier in the [[ref: VC Data Model]] corresponds to a term Relying Party (RP) in [[ref:SIOPv2]].
+[[def:Authorization Response]]
+~ OAuth 2.0 Authorization Response extended by [[ref:OIDC]] and [[ref:OpenID4VP]]
+
+[[def:Claim, Claims]]
+~ An assertion made about a subject
+
+[[def:Decentralized Identifier, DID]]
+~ An identifier that enables verifiable, decentralized digital identity as defined by the W3C specification
+  [[ref: DID Core]]. Known as a DID.
```
Comment (Sakurann) [inline spec/spec.md:96]:
```suggestion
  [[ref: DID Core]].
```

There is already `, DID` in the description

File: spec/spec.md [hunk 5]
```diff
@@ -82,9 +82,60 @@ A description to the reader on how the document is structured.
...
+
+[[def:End User]]
+~ Participant
+
+[[def:Holder]]
+~ An entity that possesses or holds verifiable credentials and can generate verifiable presentations from them
```
Comment (Sakurann) [inline spec/spec.md:102]: please add that this is vc-data-model definition

**Review by Sakurann — CHANGES_REQUESTED** [review]

**Comment by Sakurann** [comment]
I think what's confusing to me is the inconsistency: some definitions are taken from specifications directly (eg holder), some are modified (eg SIOP) and some are introduced for the purpose of this profile (eg wallet). 
we should unify the approach

**Comment by jischr** [comment]
@Sakurann thanks for the feedback. updating now to unify.

File: spec/spec.md [hunk 6]
```diff
@@ -82,9 +82,60 @@ A description to the reader on how the document is structured.
...
+[[def:Verifier]]
+~ An entity that receives one or more verifiable credential inside a verifiable presentation for processing. Synonymous
+  with the term [[ref: Relying Party (RP)]]
+
+[[def:Verification]]
+~ The process in which a [[ref:Verifier]] validates that the verifiable credential inside a verifiable presentation is authentic
+  and a timely statement of the issuer or presenter
+
+[[def:Wallet]]
+~ A credential repository that acts as a [[ref:Self Issued OpenID Provider (SIOP)]]
```
[the remark this answers is not in the document]
Comment (jischr) [reply spec/spec.md:136]: i can update to match this one.

**Review by quartzjer — APPROVED** [review]

**Comment by jischr** [comment]
@Sakurann good to go? you have to approve because of your "change requested"

File: spec/spec.md (hunk 3 above)
Comment (jischr) [reply spec/spec.md:84]: done

File: spec/spec.md [hunk 7]
```diff
@@ -83,9 +83,59 @@ A description to the reader on how the document is structured.
 
 ## Terminology
 
-Common terms in this document come from the specifications listed in (#reference).
+This section defines common terms from the profile for the reader's ease and convenience. These, as well as other useful terms, can be found in more detail within the specifications from which they originate. See [[ref:References]].
```
Comment (Sakurann) [inline spec/spec.md:86]:
```suggestion
This section consolidates in one place common terms used across open standards that this profile consists of. For the details of these, as well as other useful terms, see text within each of the specification listed in [[ref:References]].
```

**Comment by Sakurann** [comment]
Yes! Thanks a lot, Jen!

**Review by Sakurann — APPROVED** [review]

**Pull request merged**